packages/teleterm/src/services/config/providers/keyboardShortcutsConfigProvider.ts

```diff
@@ -50,7 +50,7 @@ export const keyboardShortcutsConfigProvider: ConfigServiceProvider<KeyboardShor
         'toggle-identity': 'Command-I',
       };
 
-      const linuxShortcuts: KeyboardShortcutsConfig = {
+      const windowsAndLinuxShortcuts: KeyboardShortcutsConfig = {
```

Comment:
I just noticed that the shortcut for `tab-next` is `Ctrl-PageDown`. Shouldn't it be `Ctrl-Tab`? AFAIK that's what browsers use on those systems and that's what I found in [help articles for Edge](https://support.microsoft.com/en-us/microsoft-edge/keyboard-shortcuts-in-microsoft-edge-50d3edab-30d9-c7e4-21ce-37fe2713cfad) too.

Reply:
I'll address it in the next PR as discussed.